HEAD no longer builds without rtadv

We use this to build FRR RPMs:

```rpmbuild -bb --define "dist .el7" --define "release_rev 1234" --with cumulus --define "with_rtadv 0" redhat/frr.spec```

90446e3c3310001a7b84017fa4b237ea7914f45e works just fine with this, but 4c5122452ae496fcba0ce26883370fd86642b756 throws errors about rtadv symbols:

```
zebra/interface.o: In function `shutdown_if':
 /root/rpmbuild/BUILD/frr-7.4-dev-vultr/zebra/interface.c:1974: undefined reference to `rtadv_stop_ra'
 zebra/main.o: In function `sigint':
 /root/rpmbuild/BUILD/frr-7.4-dev-vultr/zebra/main.c:148: undefined reference to `rtadv_stop_ra_all'
 zebra/connected.o: In function `connected_add_ipv6':
 /root/rpmbuild/BUILD/frr-7.4-dev-vultr/zebra/connected.c:495: undefined reference to `rtadv_add_prefix'
 zebra/connected.o: In function `connected_delete_ipv6':
 /root/rpmbuild/BUILD/frr-7.4-dev-vultr/zebra/connected.c:542: undefined reference to `rtadv_delete_prefix'
 collect2: error: ld returned 1 exit status
```

This is an issue with stable/7.3 as well (c0038fca9b4b736a59d6ed786c12ac0c8a2909ac) although there are fewer undefined references:

<pre>
 zebra/interface.o: In function `shutdown_if':
 /root/rpmbuild/BUILD/frr-7.3-vultr/zebra/interface.c:1981: undefined reference to `rtadv_stop_ra'
 zebra/main.o: In function `sigint':
 /root/rpmbuild/BUILD/frr-7.3-vultr/zebra/main.c:148: undefined reference to `rtadv_stop_ra_all'
 collect2: error: ld returned 1 exit status
</pre>

--- a/zebra/rtadv.c
+++ b/zebra/rtadv.c
@@ -2399,4 +2399,15 @@ void rtadv_cmd_init(void)
 {
 	/* Empty.*/;
 }
+
+void rtadv_stop_ra(struct interface *ifp)
+{
+	/* Empty.*/;
+}
+
+void rtadv_stop_ra_all(void)
+{
+	/* Empty.*/;
+}
+
 #endif /* HAVE_RTADV */

